apps/machine/zynqmp/zynqmp_linux_r5_proc.c

```diff
@@ -88,6 +88,26 @@ zynqmp_linux_r5_proc_init(struct remoteproc *rproc,
     prproc->shm_io = metal_device_io_region(dev, 0);
     if (!prproc->shm_io)
         goto err2;
+
+#ifdef RPMSG_NO_IPI
+	/* Get poll shared memory device */
+	ret = metal_device_open(prproc->shm_poll_bus_name,
+				prproc->shm_poll_name,
+				&dev);
+	if (ret) {
+		fprintf(stderr,
+				"ERROR: failed to open shm poll device: %d.\r\n",
+				ret);
```

Comment:
This could be written on a single line 
fprintf(stderr,	"ERROR: failed to open shm poll device: %d.\r\n", ret);

Reply:
added new commit for this